### PR TITLE
SPEC-829 - Timestamp is an unsigned 64-bit integer.

### PR DESCRIPTION
**Pull request by llvtt**

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -185,7 +189,7 @@ <h2 class="nojs">Specification Version 1.1</h2>
                   <tr>
                     <td class="fix nt"></td>
                     <td class="fix op">|</td>
-                    <td class="fix ex">"\x11" e_name int64</td>
+                    <td class="fix ex">"\x11" e_name uint64</td>
```
Comment (llvtt) [inline spec.html]: It's documented on the same page: https://github.com/mongodb/bsonspec.org/pull/46/files/e611fcd478fd2fccba84628f4bc16e8036ddb03c#diff-3540caefa502006d8a33cb1385720803R300

Comment (behackett) [inline spec.html]: Ah, right, in the notes. OK, LGTM.

Comment (llvtt) [inline spec.html]: Thanks for the review, but I can't merge this PR. Not sure who has permission to do so.

**Comment by behackett** [comment]
@kay-kim can you merge this?

**Comment by kay-kim** [comment]
merged!

**Pull request closed without merging**